File: CH2-modules/EX1-kilometer-converter/kilometerConverter.js
const input = require("prompt-sync")();

function converts(kilometer) {
  let miles;
  miles = kilometer.value * 0.6214;
  console.log("Distance in miles :" + miles);
}

function main() {
  let miles = {
    value: 0,
  };

  miles.value = Number(input("Enter a Distance in kilometers :"));

  converts(miles);
}

// Entry Point
main();

File: CH2-modules/EX1-kilometer-converter/kilometerConverter.cpp
#include <iostream>
using namespace std;

void converts(double &kilometer)
{
    double miles;

    miles = kilometer * 0.6214;

    cout << "Distance in miles :" << miles << endl;
}

int main()
{
    double kilometer;

    cout << "Enter a Distance in kilometers :";
    cin >> kilometer;

    converts(kilometer);
}